Cesium.Runtime/Cesium.Runtime.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
What do you think of targeting netstandard2.0 in our runtime library? I intend to support .NET Framework (not all versions, but at least the most recent ones) if it's not too tedious.

Reply:
Will change. I think that would be fine.

Reply:
If we insist on having char as UTF-8 char, then for netstandard2.0 I cannot use `Marshal.PtrToStringUTF8` and we have to manually convert string to UTF8.

Reply:
Hm, alright. For the time being, let's live with `net6.0`, then.

Reply:
I have both `netstandard2.0` version and `net6.0` version. Should I remove `netstandard2.0` ?

Reply:
I see, you've implemented the method yourself.

Let's leave this in the current state, then. `netstandard2.0;net6.0` looks okay to me for now.